crypto_ccm.go

```diff
@@ -9,23 +9,27 @@ import (
 	"github.com/pion/dtls/v2/internal/crypto/ccm"
 )
 
+type cryptoCCMTagLen int
```

Comment:
We can change this later, but why is this its own type? Looking at the code, everywhere it's used we do `int(cryptoCCMTagLen)`. This seems a bit pointless, if all we ever do with it is convert it back to `int` before we can use it.

Reply:
I want to provides meaningful values. Int type nothing says which values are supported.